timemachine/datasets/__init__.py

```diff
@@ -0,0 +1,3 @@
+from timemachine.datasets.utils import fetch_freesolv
+
+__all__ = [fetch_freesolv]
```

Comment:
nit: I've found this a bit unclear in the past, but I think `__all__` is supposed to be a list of strings? ([reference](https://docs.python.org/3/reference/simple_stmts.html#:~:text=the%20public%20names%20defined%20by%20a%20module%20are%20determined%20by%20checking%20the%20module%E2%80%99s%20namespace%20for%20a%20variable%20named%20__all__%3B%20if%20defined%2C%20it%20must%20be%20a%20sequence%20of%20strings%20which%20are%20names%20defined%20or%20imported%20by%20that%20module.))